xml/admin_ceph_gateway.xml

```diff
@@ -3279,365 +3212,211 @@ rgw_zone=us-east-1</screen>
+           configuration, the secondary zone is a read-only zone. Remove the
+           <option>--read-only</option> status to allow the zone to receive
+           write operations. For example:</para>
+<screen>&prompt.cephuser;radosgw-admin zone modify --rgw-zone=<replaceable>ZONE-NAME</replaceable> --master --default \
+                                                   --read-only=false
+</screen>
+       </step>
+       <step>
+         <para>Update the period to make the changes take effect:</para>
+<screen>&prompt.cephuser;radosgw-admin period update --commit</screen>
+       </step>
+       <step>
+         <para>Restart the &cogw;:</para>
+<screen>&prompt.cephuser;systemctl restart ceph-radosgw@rgw.`hostname -s`</screen>
+       </step>
+     </procedure>
+     <para>If the former master zone recovers, revert the operation.</para>
+     <procedure>
+       <step>
+         <para>From the recovered zone, pull the latest realm configuration
+           from the current master zone.</para>
+<screen>&prompt.cephuser;radosgw-admin realm pull --url=<replaceable>URL-TO-MASTER-ZONE-GATEWAY</replaceable> \
+                           --access-key=<replaceable>ACCESS-KEY</replaceable> --secret=<replaceable>SECRET</replaceable>
+</screen>
+       </step>
+       <step>
+         <para>Make the recovered zone the master and default zone:</para>
+<screen>&prompt.cephuser;radosgw-admin zone modify --rgw-zone=<replaceable>ZONE-NAME</replaceable> --master --default</screen>
+       </step>
+       <step>
+         <para>Update the period to make the changes take effect:</para>
+<screen>&prompt.cephuser;radosgw-admin period update --commit</screen>
+       </step>
+       <step>
+         <para>Restart the &cogw; in the recovered zone:</para>
+<screen>&prompt.cephuser.ogw;systemctl restart ceph-radosgw@rgw.`hostname -s`</screen>
+       </step>
+       <step>
+         <para>If the secondary zone needs to be a read-only configuration, update the secondary zone:</para>
+<screen>&prompt.cephuser;radosgw-admin zone modify --rgw-zone=<replaceable>ZONE-NAME</replaceable> --read-only</screen>
+       </step>
+       <step>
+         <para>Update the period to make the changes take effect:</para>
+<screen>&prompt.cephuser;radosgw-admin period update --commit</screen>
+       </step>
+       <step>
+         <para>Restart the &cogw; in the secondary zone:</para>
+<screen>&prompt.cephuser.ogw;systemctl restart ceph-radosgw@rgw.`hostname -s`</screen>
+       </step>
+     </procedure>
+   </sect2>
+   <sect2 xml:id="ceph-rgw-single-site-multi">
+     <title>Migrating a Single Site System to Multi-Site</title>
+     <para>To migrate from a single site system with a default &zonegroup;
+       and zone to a multi site system, use the following steps:</para>
+     <procedure>
+       <step>
+         <para>Create a realm. Replace <replaceable>NAME</replaceable> with
+         the realm name:</para>
```

Comment:
In this procedure, there is a mix of instruction descriptions ending with colons (like this one) and ending in periods (i.e. full stops). It should be consistent. 

For a direct command, a colon is fine...

`Do this:`

When the direct command is followed by clarification, it is better without...

`Do this. That means foo the bar with the baz.`

Since we have a mixture, I suggest replacing all terminating colons with periods.